website/content/docs/developers/compiling/linux.md

```diff
@@ -21,10 +21,22 @@ $ sudo apt-get upgrade -y
 ### Step 2 --- Install required libraries
 
 ```bash
-$ sudo apt-get install git build-essential automake cmake libtool-bin zip \
+$ sudo apt-get install git build-essential automake libtool-bin zip \
   libunwind-setjmp0-dev zlib1g-dev unzip pkg-config python-setuptools -y
 ```
 
+In addition, you'll also need to install [cmake](https://cmake.org/) version
+2.2.3 (this is *not* the default version installed by `apt-get`):
```

Comment:
actually the better way to do this would be to add edison backports and just do apt-get install.